### PR TITLE
Logout button and functionality

### PR DESCRIPTION
**Pull request by danielshmidov**
https://i.imgur.com/jhdzHoD.jpeg

### CODE REVIEW

File: src/components/Testing.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export default props => {
+  return <p>Hi {props.name}</p>
+}
```
Comment (acchiang) [inline src/components/Testing.js:5]: ohp we should delete this

**Review by acchiang — CHANGES_REQUESTED** [review]
also pls hide button when user logged out

**Review by Ciuffi — APPROVED** [review]
Looks good to me!

**Review by acchiang — APPROVED** [review]
lol @jackyzha0 sorry for the merge conflicts

**Comment by acchiang** [comment]
Screen Shot 2020-12-03 at 8 45 32 PM
we changed logout redirect to go to home (cuz it’s weird for someone to logout and immediately go to login page without any way to go to the home page in the UI imo) and added this Skip for now link in login page so users can still access home page when they’re not logged in

**Pull request merged**